app/src/components/modal/modal_your_connection/index.tsx

```diff
@@ -164,6 +177,24 @@ const ConnectionModalNavigation = styled(ModalNavigation as any)`
   margin-bottom: 16px;
 `
 
+const IconSettingsWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 65px;
```

Comment:
not good idea to set margins statically like this....Instead you can se left margin to auto and set the right one how much is needed to be between it and the change button....